airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/StatusView.tsx

```diff
@@ -87,24 +125,8 @@ const StatusView: React.FC<StatusViewProps> = ({ connection, frequencyText }) =>
             <FormattedMessage id={"sources.syncHistory"} />
             {connection.status === ConnectionStatus.ACTIVE && (
               <div>
-                <Button onClick={() => setIsModalOpen(true)}>
-                  <FormattedMessage id={"connection.resetData"} />
-                </Button>
-                <SyncButton
-                  disabled={!allowSync}
-                  isLoading={isLoading}
-                  wasActive={showFeedback}
-                  onClick={() => startAction({ action: onSync })}
-                >
-                  {showFeedback ? (
-                    <FormattedMessage id={"sources.syncingNow"} />
-                  ) : (
-                    <>
-                      <TryArrow icon={faRedoAlt} />
-                      <FormattedMessage id={"sources.syncNow"} />
-                    </>
-                  )}
-                </SyncButton>
+                {wrapWithTooltip(resetDataBtn)}
+                {wrapWithTooltip(syncNowBtn)}
```

Comment:
Again a more stylistic comment:
```
<WithTooltip>
{syncNowBtn}
</WithTooltip>
```
Or just putting all the jsx in the render.

Reply:
Ok, sure. This is probably the best solution that we can implement with the current implementation of ToolTip.
I was wondering why we can't control tooltip appearing via props like 
`<ToolTip control={component} show={isAtLeastOneJobRunningOrPending}> ... />`